packages/cdai/src/index.scss

```diff
@@ -9,5 +9,6 @@
 
 @import 'components/IdeButton/IdeButton';
 @import 'components/IdeEmptyState/IdeEmptyState';
+@import 'components/IdeEmptyState/IdeImporting';
```

Comment:
shouldn't that be `'components/IdeImporting/IdeImporting'` ?

Reply:
yep, you're right 👍

Reply:
```suggestion
@import 'components/IdeImporting/IdeImporting';
```